fuel-vm/src/gas/default_gas_costs.rs

```diff
@@ -1,24 +1,24 @@
 use super::*;
 /// File generated by fuel-core: benches/src/bin/collect.rs:440. With the following git hash
-pub const GIT: &str = "21287fb7d82b05e6f4d94bae2f67ec8e74fd9ca0";
+pub const GIT: &str = "8d16a9266c2c6945b320bfc959633c7c57cdfa10";
 pub fn default_gas_costs() -> GasCostsValues {
     GasCostsValues {
         add: 1,
         addi: 1,
         aloc: 1,
         and: 1,
         andi: 1,
-        bal: 155,
+        bal: 21,
         bhei: 1,
         bhsh: 1,
         burn: 35,
         cb: 2,
         cfei: 1,
         cfsi: 1,
-        croo: 163,
+        croo: 28,
         div: 1,
         divi: 1,
-        ecr: 1601,
+        ecr: 1703,
```

Comment:
Interesting to see `ecr` go up by so much while most others went down

Reply:
It varies by about 100 each time I run the test, we likely need larger sample sizes but this good enough approximation for now.